src/renderer/shared/core/types/account.ts

```diff
@@ -23,7 +23,7 @@ export type BaseAccount = AbstractAccount & {
 // };
 
 export type ChainAccount = BaseAccount & {
-  baseId: BaseAccount['id'];
+  baseId?: BaseAccount['id'];
```

Comment:
This is a big change, why it became optional?

Reply:
We need it for derived accounts, they don't have base account at all, only chain accounts